### ISSUE TITLE
[Suggestion]: Pandora+ execution without GUI

### ISSUE THREAD

**Issue by PinyaGit** [Monitor144hz/Pandora-Behaviour-Engine-Plus#114]
### Version

1.1.6

### Provide Instructions on Reproducing the Problem

Couldn't find a Label on the suggestions, so hopefully I won't be hit with bricks.

FNIS had a key InstantExecute=1, which allowed to automatically start animation generation by starting FNIS via MO2. 
Pandora has a file ActiveMods.txt that remembers previously activated mods perfectly.

Question, is it possible to implement a way to start Pandora without GUI? So that after starting via MO2 it generates animations and turns itself off. Thanks in advance.

### Provide Relevant Files

[2.4.1 OStim Pandora Output.zip](https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/files/14300181/2.4.1.OStim.Pandora.Output.zip)

### Contributing To Development

- [X] I read this project's [Contributing To Development](https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/wiki/Contributing-To-Development)

**Comment by Monitor144hz** [comment]
@PinyaGit technically possible yes, but is there a valid use case for this? Users only rerun their engine when they change their behavior mod setup.

**Comment by elbadcode** [comment]
it looks like he curates a modpack so he's possibly just hoping for a way to automate that part of the process without having to teach a new interface to users with questionable literacy or provide every possible combo of completed patchers if there are multiple profiles. I don't see automated patching as any more foolproof than simply shipping a few pregenerated outputs but it would at least look cleaner in mo2.

If its not difficult to add I kind of would like to see pandora ship with a headless CLI mode that accepts a few params, even just something as simple as specifying an ActiveMods.txt to read from in lieu of sorting in the GUI since that opens up a few workflow possibilities. Could expose some debug options that don't make sense to be in the GUI but have some value, e.g. whether to update existing behavior patches or do a clean generation, or maybe to target a specific behavior project for individual patch generation if that makes sense? Could be useful for custom projects down the line perhaps, but even now I could see some small benefit to being able to A/B test some patch lists without needing to reload the MO2 VFS. Definitely a low priority feature though

**Comment by PinyaGit** [comment]
@elbadcode Yes! You're exactly right. Explaining obvious things by pressing one Launch button may seem complicated, so I want to automate such processes. Pandora+ is used (for now) on 3 profiles and each has a modified list of animations. It's no problem for me to make 3 Output folders and generate them there, however people will set their own set of animations, which will already make changes. So I want to make it as automatic as possible and the user doesn't even have to think about what to do there.

**Comment by BF3000** [comment]
It is better to launch Pandora every time we launch Skyrim. No need to launch it manually, Pandora is fast enough for this. Maybe even add scan only first and do nothing if nothing changed.

**Comment by PinyaGit** [comment]
> It is better to launch Pandora every time we launch Skyrim. No need to launch it manually, Pandora is fast enough for this. Maybe even add scan only first and do nothing if nothing changed.

Nah, keeping track of animation changes every time you start Skyrim is pointless. You don't add new animations every time you launch the game, do you? Especially running the game through SKSE, how will it run Pandora?

**Comment by Monitor144hz** [comment]
@PinyaGit does a startup argument like `-autorun` to run automatically based on ActiveMods.txt work for now? A CLI app may be coming later down the line but I plan to expand it to be a tool more so for mod authors rather than users.

**Comment by Monitor144hz** [comment]
Completed in #132

**Comment by PinyaGit** [comment]
![изображение](https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/assets/39755512/a83a7b67-7d44-45b0-98b4-250ad753753c)
![изображение](https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/assets/39755512/c0ec1f3a-5106-4158-aeee-b76641b5709a)
unfortunately, it's not working for me. ActiveMods.txt file is present

**Comment by Monitor144hz** [comment]
also keep in mind the new version with commands hasn't been pushed to release yet. you'll have to grab the build artifact if you want to try it for now
